<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icon.jpg</normaloff>images/icon.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="l_family">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>字体</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="cb_bold">
     <property name="text">
      <string>字体加粗</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="btn_ret">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QCheckBox" name="cb_italic">
     <property name="text">
      <string>斜体</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="l_pointsize">
     <property name="text">
      <string>字体像素大小</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="sb_pointsize">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QTextEdit" name="te_show">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="markdown">
      <string>三硝基甲苯TNTtnt  abcABC1234567890

</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;三硝基甲苯TNTtnt  abcABC1234567890&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="l_min">
     <property name="text">
      <string>最大间隔分钟数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="cbb_family">
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cb_new_autorun">
     <property name="text">
      <string>新建时自启</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="sb_min">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>21</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="btn_sure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="cb_del_tip">
     <property name="text">
      <string>删除确认</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="cb_underline">
     <property name="text">
      <string>下划线</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
